install/nodejs

```diff
@@ -189,6 +189,12 @@ install_nodejs()
 
         # Workaround issue with Catalina
         command_exists "npm" || brew link "node@${mac_lts_version}"
+    elif os_distro alpine; then
+
+        # nodejs precompiled binaries are against glib, and alpine needs musl
+        # for now, use whatever alpine is providing
```

Comment:
OH man. Alpine is always such a problem child.

Reply:
😭 
This could be an option for later: https://unofficial-builds.nodejs.org/